src/example-relay/src/Homepage/locations/__tests__/LocationsPaginatedRefetch.test.js

```diff
@@ -15,6 +15,7 @@ beforeEach(() => {
 
 const TestRenderer = () => (
   <QueryRenderer
+    // $FlowFixMe upgrading relay
```

Comment:
Would it be possible to use some more descriptive comment? It would be nice to at least know it's happening from 9.1.

Reply:
Changed to `// $FlowFixMe errors after upgrading to relay 9.1.0`

Reply:
Yes, much better. Now, we can find the exact suppression comment related to this version change easily (and fix it).